Tests/HummingbirdMustacheTests/SpecTests.swift

```diff
@@ -104,11 +104,13 @@ final class MustacheSpecTests: XCTestCase {
         let data = try Data(contentsOf: url)
         let spec = try JSONDecoder().decode(Spec.self, from: data)
 
-        print(spec.overview)
+        // print(spec.overview)
```

Comment:
Print statement

Reply:
That's in the tests, also it shouldn't be commented out